README.md

```diff
@@ -85,7 +85,47 @@ echo "{
 ```
 
 ### Connecting to a local graph store
-As mentioned in the introduction, it is possible to connect `graph-notebook` to a graph database running on your local machine. An example being Gremlin Server. There are additional instructions regarding the use of local servers in the `additional-databases` folder.
+As mentioned in the introduction, it is possible to connect `graph-notebook` to a graph database running on your local machine, an example being Gremlin Server. There are additional instructions regarding the use of local servers in the `additional-databases` folder.
```

Comment:
Can you make graph-notebook a link to the directory?

Reply:
Same for additional-databases.

Reply:
Done!